docs/getting-started-guides/juju.md

```diff
@@ -260,33 +290,58 @@ project on github.com:
 
 ### Cloud compatibility
 
-Juju runs natively against a variety of public cloud providers. Juju currently
-works with [Amazon Web Service](https://jujucharms.com/docs/stable/config-aws),
-[Windows Azure](https://jujucharms.com/docs/stable/config-azure),
-[DigitalOcean](https://jujucharms.com/docs/stable/config-digitalocean),
-[Google Compute Engine](https://jujucharms.com/docs/stable/config-gce),
-[HP Public Cloud](https://jujucharms.com/docs/stable/config-hpcloud),
-[Joyent](https://jujucharms.com/docs/stable/config-joyent),
-[LXC](https://jujucharms.com/docs/stable/config-LXC), any
-[OpenStack](https://jujucharms.com/docs/stable/config-openstack) deployment,
-[Vagrant](https://jujucharms.com/docs/stable/config-vagrant), and
-[Vmware vSphere](https://jujucharms.com/docs/stable/config-vmware).
+Juju is cloud agnostic and gives you a consistent experience across different
+cloud providers. Juju supports a variety of public cloud providers: [Amazon Web Service](https://jujucharms.com/docs/2.0/help-aws),
+[Microsoft Azure](https://jujucharms.com/docs/2.0/help-azure),
+[Google Compute Engine](https://jujucharms.com/docs/2.0/help-google),
+[Joyent](https://jujucharms.com/docs/2.0/help-joyent),
+[Rackspace](https://jujucharms.com/docs/2.0/help-rackspace), any
+[OpenStack cloud](https://jujucharms.com/docs/2.0/clouds#specifying-additional-clouds),
+and
+[Vmware vSphere](https://jujucharms.com/docs/2.0/config-vmware).
 
 If you do not see your favorite cloud provider listed many clouds with ssh
 access can be configured for
-[manual provisioning](https://jujucharms.com/docs/stable/config-manual).
+[manual provisioning](https://jujucharms.com/docs/2.0/clouds-manual).
```

Comment:
Yeah, lets remove that for now, then this should be ready to go. Thanks for your contributions.